### DISCUSSION TITLE
Can Trivy scan for sensitive information within JAR files?

### DISCUSSION THREAD

**Discussion by m1ssya** [aquasecurity/trivy#7719] [Q&A]
### Question

Can Trivy scan for sensitive information within JAR files? Because configuration files inside JAR files may also leak sensitive information.

### Target

None

### Scanner

Secret

### Output Format

None

### Mode

None

**Comment by knqyf263** [comment]
Trivy currently supports only plaintext for secret scanning except for compiled python files.
https://aquasecurity.github.io/trivy/v0.56/docs/scanner/secret/
